tests/lib/managers/collaboration-whitelists-test.js

```diff
@@ -0,0 +1,427 @@
+/**
+ * @fileoverview Collaboration Whitelists Manager Tests
+ */
+'use strict';
+
+// ------------------------------------------------------------------------------
+// Requirements
+// ------------------------------------------------------------------------------
+var sinon = require('sinon'),
+	mockery = require('mockery'),
+	assert = require('chai').assert,
+	leche = require('leche');
+
+var BoxClient = require('../../../lib/box-client');
+
+// ------------------------------------------------------------------------------
+// Helpers
+// ------------------------------------------------------------------------------
+var sandbox = sinon.sandbox.create(),
+	boxClientFake = leche.fake(BoxClient.prototype),
+	CollaborationWhitelists,
+	collaborationWhitelists,
+	testQS = { testQSKey: 'testQSValue' },
+	testParamsWithQs,
+	COLLABORATION_WHITELIST_ID = '1234',
+	USER_COLLABORATION_WHITELIST_ID = '5678',
+	USER_ID = '12345678',
+	MODULE_FILE_PATH = '../../../lib/managers/collaboration-whitelists';
+
+// ------------------------------------------------------------------------------
+// Tests
+// ------------------------------------------------------------------------------
+
+describe('CollaborationWhitelists', function() {
+
+	before(function() {
+		// Enable Mockery
+		mockery.enable({ useCleanCache: true });
+		// Register Mocks
+		mockery.registerAllowable('../util/url-path');
+		mockery.registerAllowable(MODULE_FILE_PATH);
```

Comment:
This should take a second param: `mockery.registerAllowable(MODULE_FILE_PATH, true);` to make sure that just the module under test gets reloaded for every test (to clear any accidental state between tests)